terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/time/dateAndTimeDisplay.jsp

```diff
@@ -0,0 +1,106 @@
+
```

Comment:
- [ ] Since this is a JSP that was created when Tiles existed, add a header just like any other JSP.

Reply:
Added a header.
f491dbc59b57149b22f6750e17b284c9c61a3353